### PR TITLE
Cortex 1.9 Adjustments

### PR DESCRIPTION
**Pull request by ts-mini**
Based on some deprecations in the [release notes for 1.9](https://github.com/cortexproject/cortex/releases/tag/v1.9.0) this should help solve: https://github.com/cortexproject/cortex-helm-chart/issues/153

### CODE REVIEW

**Comment by cabrinha** [comment]
@ts-mini maybe you could try adding @khaines as a reviewer?

**Comment by khaines** [comment]
@ts-mini @cabrinha - Unfortunately, I'm no longer a maintainer for the cortex project org (lack of time). I don't have the permission access to merge this. Please raise this in the #cortex slack channel as needing review & merge. 

overall it looks good to me since it's a straightforward change.

File: values.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@ config:
   auth_enabled: false
   api:
     prometheus_http_prefix: '/prometheus'
+    response_compression_enabled: true
```
Comment (ShuzZzle) [inline values.yaml:49]: this is not currently enabled by default. As this has an effect on CPU performance I don't know if we should. Whats your take on this @cabrinha ?

**Review by ShuzZzle — COMMENTED** [review]
LGTM but unsure about enabling api response compression by default

File: values.yaml (hunk 1 above)
Comment (cabrinha) [reply values.yaml:49]: I think it's fine to enable compression by default. Which component does this affect?

Comment (ShuzZzle) [reply values.yaml:49]: Too quote the docs:
>   Use GZIP compression for API responses. Some endpoints serve large YAML or
>   JSON blobs which can benefit from compression.

So I am guessing every cortex API Response

Comment (cabrinha) [reply values.yaml:49]: Let's use compression by default. Users can deal with the CPU on their end with requests / limits. I bet the impact to CPU is minimal.

**Comment by ShuzZzle** [comment]
@ts-mini Can you please update the CHANGELOG.md under the master / unreleased section?

> [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 * [ENHANCEMENT] Expose `client_max_body_size` config for nginx max request body size #137
 * [ENHANCEMENT] Adding option to add custom headers (ex. X-Scope-OrgID) to NGINX from values.yaml (key `nginx.config.setHeaders`). #127
+* [BUGFIX] Replacing deprecated (in Cortex 1.9) configuration with appropriate replacements (max_look_back_period -> max_query_lookback & compress_responses -> resposne_compression_enabled). #154
```
Comment (ShuzZzle) [inline CHANGELOG.md:7]: resposne_compression_enabled

tiny typo

Comment (ts-mini) [reply CHANGELOG.md]: :facepalm:

**Review by ShuzZzle — APPROVED** [review]

**Pull request merged**